### ISSUE TITLE
Sorry, your account is not authorised for any further contract purchases?

### ISSUE THREAD

**Issue by buckman1974** [binary-com/binary-bot#1484]
I am suddenly getting this error today ?

"Sorry, your account is not authorised for any further contract purchases."

So no bot will run? this is on a Virtual Account for testing, so guess it must be the same with a Real Account too?

Any ideas what this is and if everyone is getting this today?

**Comment by aarimm** [comment]
Hey @buckman1974,

Could you open a ticket through our help system, click the big orange question mark on our main website or on the Binary Bot website and we'll get back to you.

Regards,
Aaron

**Comment by buckman1974** [comment]
Thanks @aarimm  - I have big orange question mark on your main website, and typed in my problem, the page "Help Centre" is just loading forever????

**Comment by anton-binary** [comment]
Dear @buckman1974,

According to new [UKGC](https://www.gamblingcommission.gov.uk/home.aspx) requirement we must age verify **all** accounts (new and existing) including virtual ones. This requirement applied on 7th of May for MX and MLT accounts. You could have already received an email clarifying on that (if not then you'll receive shortly).

What you can do is to upgrade to real account in case you have't done that yet and/or authenticate it (select Settings -> Profile -> Authenticate).

**Comment by buckman1974** [comment]
Thank you @anton-binary and @aarimm 

My problem was that i thought i had been hacked by some other Binary.com user trying to sell strategies and maybe had one of my virtual account numbers.

Basically I set up a Real Account a while back (which obviously came with a Virtual Account). To practice at low balances, I used my other email addresses to anonymously use ONLY the Virtual Accounts for practice. So I had : 

1. Real Account (and Virtual Account)
2. Virtual Account with new email
3. Virtual Account with new email
4. Virtual Account with new email
5. Virtual Account with new email

I now went and verified my Real Account finally (as I was still using its Virtual Account) and was all verified successfully with Binary.com. But since then, the unrelated virtual accounts above all stopped working with the 
**"Sorry, your account is not authorised for any further contract purchases."**

Is this normal? or is this hacked?

If its normal as I have just decided to finally verify my main account successfully, shall i just delete all my other practice accounts as this is standard procedure once verifying my main account?

**Comment by anton-binary** [comment]
Hey @buckman1974 ,

According to new regulation **all** accounts (real and virtual) should be aged verified, so in case if you have multiple virtual accounts registered to different emails, you have to age verified each and every account, i.e. upgrade them to real accounts and proceed with authentication. That's the consequence of new regulation.

Regards,
Anton

**Comment by buckman1974** [comment]
Ok thanks @anton-binary , so suddenly all of my extra 'virtual accounts' on different emails, have been triggered now to not work.

Does this coincide with my main account being verified, and its cancelled out all the others?

At least I know its NOT down to any scam or hack into my account?

So if this is the case, then I can delete my other virtual accounts now, and stick with the main (verified) account for Virtual and Real ?

**Comment by anton-binary** [comment]
Hey @buckman1974 ,
>Does this coincide with my main account being verified, and its cancelled out all the others?
Virtual accounts registered to different emails are treated as unrelated unless you upgrade them to real ones. So, verification of one of your virtual accounts does not impact other accounts registered to different emails.

>So if this is the case, then I can delete my other virtual accounts now, and stick with the main (verified) account for Virtual and Real ?
Correct, you can stick with the main (verified) account for Virtual and Real.
In case you want to delete the your other virtual accounts, please rise a ticket using our Help Center. Please list accounts and provide the reason for deletion.

Regards,
Anton.

**Comment by buckman1974** [comment]
> 
> Virtual accounts registered to different emails are treated as unrelated unless you upgrade them to real ones. So, verification of one of your virtual accounts does not impact other accounts registered to different emails.
> 

So @anton-binary why has this suddenly happened when all of my other Virtual accounts with different emails, were working fine?
My asking, and worry is that a hacker has gotten my details, and all my other accounts seem now to be suspended with this error.

My Main account seems fine, but this error to the other accounts has only happened in the last few days, when I decided to verify (successfully) my main account at last.

Is this error to the other accounts happening because of this single verification?

Or have these accounts been hacked in some way?

**Comment by anton-binary** [comment]
Hey @buckman1974 ,
TL;DR: 
**That's the consequence of new regulation.** Every account now needs age verification.
This is *new requirement* of UKGC and it applied **on 7th of May, 2019** ago. 
[https://www.gamblingcommission.gov.uk/PDF/AV-CI-Consultation-responses-Feb-2019.pdf](https://www.gamblingcommission.gov.uk/PDF/AV-CI-Consultation-responses-Feb-2019.pdf)

If you have multiple virtual accounts registered to different emails, we are unable to link them to the account you call the "Main" unless you upgrade them to Real. So, these accounts represent **different** users for us. Thus, we are required by UKGC to request age verification for all of them and that's why they are suspended even though you have already verified your "Main" account because, as i said, there's no link between your "Main" account and other virtual accounts registered to different emails.

>My asking, and worry is that a hacker has gotten my details, and all my other accounts seem now to be suspended with this error.

There has been no leak of you account details from our side. Accounts are suspended due to the UKGC requirement. However, in case if you want more secure login please check 2 factor authentication feature. https://www.binary.com/en/user/security/two_factor_authentication.html

Hope that finally clarifies the situation.

Regards,
Anton

**Comment by buckman1974** [comment]
Ok thanks @anton-binary it looks like the consequence of new regulation happened just at the same time as I was successfully verified then.

So even if you now have a virtual account, it now needs age verification due to this new UKGC requirement?

It seems like it happened together which is why suddenly my worry of any hacking or scam account

**Comment by anton-binary** [comment]
Hey @buckman1974 ,
>So even if you now have a virtual account, it now needs age verification due to this new UKGC requirement?

correct
